src/preprocessing-service/preprocess.py

```diff
@@ -0,0 +1,95 @@
+# Standard Library
+import asyncio
+import logging
+
+# Third Party
+import pandas as pd
+from elasticsearch import AsyncElasticsearch
+from elasticsearch.helpers import async_streaming_bulk
+from masker import LogMasker
+from nats_wrapper import NatsWrapper
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
+
+ES_ENDPOINT = "https://opendistro-es-client-service.default.svc.cluster.local:9200"
+
+es = AsyncElasticsearch(
+    [ES_ENDPOINT],
+    port=9200,
```

Comment:
please add the port as an ENV variable in the yaml file as well.

Reply:
port not necessary in yaml